packages/story-editor/src/elements/media/scalePanel.js

```diff
@@ -22,10 +22,11 @@ import styled from 'styled-components';
 import { _x } from '@googleforcreators/i18n';
 import { Slider } from '@googleforcreators/design-system';
 import { BG_MIN_SCALE, BG_MAX_SCALE } from '@googleforcreators/animation';
+import { InOverlay } from '@googleforcreators/moveable';
+
 /**
  * Internal dependencies
  */
-import InOverlay from '../../components/overlay';
```

Comment:
This is one place where the overlay doesn't seem to be related to Moveable -- is it still ok to move everything related to Overlay to the moveable packages, even though it's used in other cases, too?

Reply:
`InOverlay` is being used in three places in `story-editor`. Did you mean we should move the entire `ScalePanel`, `LibraryMoveable` and `SelectionCanvas` components to `movable` package?

`LibraryMoveable` btw, has a lot of internal dependencies to deal with.

https://github.com/GoogleForCreators/web-stories-wp/blob/a354b43c28b49b5b5cb83fd465c618d6278ab052/packages/story-editor/src/components/library/panes/shared/libraryMoveable.js#L26-L41

Reply:
> Did you mean we should move the entire ScalePanel, LibraryMoveable and SelectionCanvas components to movable package?

I was just wondering if it makes sense to move the whole overlay part to Moveable package at all even though it's not directly for Moveable only (as opposed to keeping it in the editor package). But I guess that's the best option here.